playground/cdk/CdkSample.ApiService/Program.cs

```diff
@@ -2,16 +2,29 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Azure.Storage.Blobs;
+using Microsoft.EntityFrameworkCore;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
 
 builder.AddAzureBlobService("blobs");
+builder.AddSqlServerDbContext<SqlContext>("sqldb");
 
 var app = builder.Build();
 
-app.MapGet("/", async (BlobServiceClient bsc) =>
+app.MapGet("/", async (BlobServiceClient bsc, SqlContext context) =>
+{
+    return new
+    {
+        blobFiles = await TestBlobStorageAsync(bsc),
+        sqlRows = await TestSqlServerAsync(context)
+    };
+});
```

Comment:
This is gonna be the slowest endpoint ever.

Reply:
Having an API do something with SQL and blobs is pretty common ;)